### PR TITLE
EZP-29461: Missing configuration of the Multi-file upload 

### PR DESCRIPTION
**Pull request by adamwojs**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-29461
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | ?
| Tests pass?   | yes
| Doc needed?   | yes
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Moved missing configuration of the Multi-file upload from https://github.com/ezsystems/ezplatform-multi-file-upload.

#### Checklist:
- [X] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/DependencyInjection/Configuration/Parser/Module/MultiFileUpload.php [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+namespace EzSystems\EzPlatformAdminUiBundle\DependencyInjection\Configuration\Parser\Module;
+
+use eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\AbstractParser;
+use eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\SiteAccessAware\ContextualizerInterface;
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+
+class MultiFileUpload extends AbstractParser
+{
+    /**
+     * {@inheritdoc}
```
Comment (ViniTou) [inline src/bundle/DependencyInjection/Configuration/Parser/Module/MultiFileUpload.php]: full example configuration could be added into docblock

**Comment by lserwatka** [comment]
@adamwojs could you rebase with 1.2 branch?

**Comment by adamwojs** [comment]
@lserwatka done

File: src/bundle/Resources/config/ezplatform_default_settings.yml [hunk 2]
```diff
@@ -47,3 +47,43 @@ parameters:
...
+          content_type_identifier: file
+          content_field_identifier: file
+          name_field_identifier: name
+
+    ezsettings.default.multifile_upload_module.fallback_content_type:
+        content_type_identifier: file
+        content_field_identifier: file
+        name_field_identifier: name
+
+    ezsettings.default.multifile_upload_module.max_file_size: 5242880
```
Comment (webhdx) [inline src/bundle/Resources/config/ezplatform_default_settings.yml]: Above in the semantic configuration definition there is 64MB as a default value. Please streamline it.

**Comment by andrerom** [comment]
Any chance this will make it for 2.5? Or is it not relevant anymore?

**Comment by adamwojs** [comment]
Closing as obsolete

**Pull request closed without merging**